Description: Fix spelling errors found by lintian
 Lintian complains about a common spelling error in libmemcached, this patch
 fixes it.
Author: Michael Fladischer <FladischerMichael@fladi.at>
Last-Update: 2012-01-16
Forwarded: no

--- a/libmemcached/io.cc
+++ b/libmemcached/io.cc
@@ -390,7 +390,7 @@
                                      void *buffer, size_t length, ssize_t& nread)
 {
   assert(memcached_is_udp(ptr->root) == false);
-  assert_msg(ptr, "Programmer error, memcached_io_read() recieved an invalid memcached_server_write_instance_st"); // Programmer error
+  assert_msg(ptr, "Programmer error, memcached_io_read() received an invalid memcached_server_write_instance_st"); // Programmer error
   char *buffer_ptr= static_cast<char *>(buffer);
 
   if (ptr->fd == INVALID_SOCKET)
